### DISCUSSION TITLE
Better view of `pnpm ls <pkg>`

### DISCUSSION THREAD

**Discussion by vjpr** [pnpm/pnpm#3322] [Ideas]
**Describe the user story**

When trying to find all the versions of a package the output is not easy to visually scan.

My specific use case was looking at what version of `metro` is used in an expo project.

Note that the key information I am looking for is nested, and repeated multiple times.

![Screen Shot 2021-04-07 at 5 10 07 pm](https://user-images.githubusercontent.com/281413/113890093-27ad8700-97c4-11eb-82f1-48b8e0e50acc.png)

**Describe the solution you'd like**

How about inverting the tree structure so it is bottom-up, and maybe some grouping by version number perhaps?

```
metro: 0.59.0 
  @react-native-community/cli 4.14.0 > react-native 0.63.2 peer > @unimodules/react-native-adapter 5.7.0 > expo 40.0.1

metro: 0.58.0 
  ...
```

~Also, for the command needed: `pnpm ls metro --depth=99`, the depth feels like we should have an easier way to insist `depth=infinity`. Maybe a flag...~

`pnpm why` seems to solve this.

**Describe the drawbacks of your solution**

Adds additional complexity that must be maintained.

**Describe alternatives you've considered**

Create a separate cli tool.

**Comment by zkochan** [comment]
Sounds good to me
